compiler/codegen/cg-expr.cpp

```diff
@@ -795,11 +795,11 @@ static
 GenRet codegenLocaleForNode(GenRet node)
 {
   node = codegenValue(node);
-  GenRet anySublocale = codegenUseGlobal("c_sublocid_any");
+  GenRet anySublocale = codegenUseGlobal("c_sublocid_none");
```

Comment:
Can you rename the variable here to avoid future confusion?

Reply:
I am removing this altogether, it was unused.